vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/ExperimentalFeatureException.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See {@literal <https://vaadin.com/commercial-license-and-service-terms>} for the full
+ * license.
+ */
+package com.vaadin.flow.component.dashboard;
+
+/**
+ * An exception which is thrown when somebody attempts to use {@link Dashboard}
+ * or {@link DashboardWidget} components without activating the associated
+ * feature flag first.
+ *
+ * @author Vaadin Ltd
+ */
+public class ExperimentalFeatureException extends RuntimeException {
+    public ExperimentalFeatureException() {
+        super("The Dashboard component is currently an experimental feature and needs to be "
+                + "explicitly enabled. The component can be enabled using the Vaadin dev-mode Gizmo, "
```

Comment:
I think this should be rephrased as "dev-mode Gizmo" has been renamed to dev tools and nowadays it is replaced with Copilot (not sure if the feature flags are available there, since I've never used it).

Reply:
Yes, this should mention Vaadin Copilot instead.

Reply:
Updated it to reflect the changes similar to other recent feature flags.